internal/pkg/template/templates/app/cf.yml

```
@@ -8,7 +8,8 @@ Metadata:
   TemplateVersion: '{{.TemplateVersion}}'
   Version: {{.Version}}
   Services:{{if not $services}} []{{else}}{{range $service := $services}}
-  - {{$service.Name}}{{end}}{{end}}
+  - Name: {{$service.Config.Name}}
+    WithECR: {{$service.Config.WithECR}}{{end}}{{end}}
```

```suggestion
    - Name: {{$service.Config.Name}}
      WithECR: {{$service.Config.WithECR}}{{end}}{{end}}
```
nit (indent within services)